python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -3452,7 +3452,7 @@ def test_error_stack_trace(self):
         self.spark.stop()
         spark = (
             PySparkSession.builder.config(conf=self.conf())
-            .config("spark.connect.jvmStacktrace.maxSize", 128)
+            .config("spark.connect.grpc.maxMetadataSize", 128)
```

Comment:
I had to restrict amount of metadata much more otherwise the test fails w/ a gRPC failure: client cannot receive big message.